galasa-managers-parent/galasa-managers-core-parent/dev.galasa.core.manager/src/main/java/dev/galasa/core/manager/internal/CoreManager.java

```diff
@@ -42,6 +47,13 @@
 	private IConfigurationPropertyStoreService cpsTest;
 	private IConfidentialTextService           ctf;
 
+	private final static Log logger = LogFactory.getLog(CoreManager.class);
+
+	private final static Pattern patternStoreVariable = Pattern.compile("<([A-z0-9_.]+)> is test property ([A-z0-9_.]+)");
+	private final static Pattern patternLogVariable = Pattern.compile("Write to log \"Hello <([A-z0-9_.]+)>\"");
```

Comment:
Write to log \"Hello <([A-z0-9_.]+)>\"
Should be 
Write to log (.*)
You then need to look for <\w+> in the group 1 text